### ISSUE TITLE
Captions with HTML formatting

### ISSUE THREAD

**Issue by nolimits4web** [joelambert/Flux-Slider#12]
I think it will be much better if we could be able to use HTML formatted text in captions. For example i want that my captions contain some heading, some small description and hyper link. But if the Flux read the caption's content from the img title attribute it is almost impossible. Of course we can use something like that : <img src="..." title="&lt;h1&gt;Some Heading&lt;/h1&gt;.." but it is not really comfortable ;)

**Comment by joelambert** [comment]
I see what you're saying, I think its an issue of simplicity vs functionality though. I _really_ like the simplicity of using `title` as its semantically correct and doesn't require extra markup.

It might be appropriate to offer both? So Flux reads from `title` unless the `rel` is set, e.g.

```
<img src="http://..." alt="" rel="advancedcaption[captionid]" />
```

Where `captionid` is the id of a DOM element to use, for example:

```
<div id="captionid">
    <h1>Header</h1>
    <p>This is some extra info with a <a href="#">link</a></p>
</div>
```

I'd be interested to hear what others think?

**Comment by nolimits4web** [comment]
Yes right, the both versions will be great, you can make some var in configuration, something like "captionsFromRel : true". By the way the "rel" attribute is already obsolete in HTML5 and the site will not be validated when using rel, it is better to use "data-..." attributes. But there is another option, i saw it somewhere ;) ... Every image (slide) has its own index number, so you can read captions from some container based on their index numbers like:

<pre>
<code>
&lt;div id=&quot;slider&quot;&gt;
    &lt;img src=&quot;img/avatar.jpg&quot; alt=&quot;&quot; /&gt;
    &lt;img src=&quot;img/ironman.jpg&quot; alt=&quot;&quot; title=&quot;Ironman Screenshot&quot; /&gt;
    &lt;a href=&quot;&quot;&gt;&lt;img src=&quot;img/imagewithlink.jpg&quot; alt=&quot;&quot; /&gt;&lt;/a&gt;
    &lt;img src=&quot;img/tron.jpg&quot; alt=&quot;&quot; /&gt;
&lt;/div&gt;

&lt;div id=&quot;captions&quot;&gt;
  &lt;div&gt;Any HTML of First caption&lt;/div&gt;
  &lt;div&gt; ... &lt;/div&gt;
  &lt;div&gt; ... &lt;/div&gt;
  &lt;div&gt;Any HTML of last caption&lt;/div&gt;
&lt;/div&gt;
</code>
</pre>

**Comment by Pixelous** [comment]
So, a year have past can we use captions with HTML formatting now? Thanks.

**Comment by joelambert** [comment]
Hi, this is not something I'm looking to add in the short term but feel free to fork the repo and add it for your own project needs.
